cmd/vclusterctl/cmd/platform/destroy.go

```diff
@@ -65,7 +65,8 @@ VirtualClusterInstances managed with driver helm will be deleted, but the underl
 	destroyCmd.Flags().BoolVar(&cmd.IgnoreNotFound, "ignore-not-found", false, "Exit successfully if platform installation is not found")
 	destroyCmd.Flags().BoolVar(&cmd.Force, "force", false, "Try uninstalling even if the platform is not installed. '--namespace' is required if true")
 	destroyCmd.Flags().BoolVar(&cmd.NonInteractive, "non-interactive", false, "Will not prompt for confirmation")
-	destroyCmd.Flags().IntVar(&cmd.TimeoutMinutes, "timeout-minutes", 5, "How long to try deleting the platform before giving up")
+	destroyCmd.Flags().IntVar(&cmd.TimeoutMinutes, "timeout-minutes", 5, "How long to try deleting the platform before giving up. May increase when removing finalizers if --remove-finalizers is used")
+	destroyCmd.Flags().BoolVar(&cmd.ForceRemoveFinalizers, "force-remove-finalizers", false, "IMPORTANT! Removing finalizers may cause unintended behaviours like leaving resources behind, but will ensure the platform is uninstalled.")
```

Comment:
Hmm, if you use `--force`, it wouldn't skip that prompt as it is now. You have to be more specific with non-interactive.

Reply:
IMO, the finalizers part is important here because of the nuance between force removing, and force removing finalizers. Removing finalizers is sometimes the opposite of a true `--force`.

Reply:
Was that last comment supposed to have something besides the quote ?

I didn't realize there was a `non-interactive` option, so that would be used in CI/OPS environments, which is what I was mostly thinking of.  And I suppose Ricardo's feature does use a finalizer, even though it's a separate feature.